cmd/create/idp/htpasswd.go

```diff
@@ -245,7 +245,8 @@ func exitHTPasswdCreate(format, clusterKey string, err error, r *rosa.Runtime) {
 func UsernameValidator(val interface{}) error {
 	if username, ok := val.(string); ok {
 		if username == ClusterAdminUsername {
-			return fmt.Errorf("username '%s' is not allowed", username)
+			return fmt.Errorf("username '%s' is not allowed. It is preserved for cluster admin creation."+
+				"Run `rosa create admin` to create user '%s'", username, username)
```

Comment:
you'll need a newline or a space after "...admin creation."

Reply:
the `--cluster` flag is also required in the example command

Reply:
fixed, thanks